### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by George2543**
### Pull Request Overview

This pull request adds the documentation of my project

### TODO or Help Wanted

Maybe this pull request still needs modifications.

### Author

Signed-off-by: Radu George-Alexandru <georgeradu642@gmail.com>

### CODE REVIEW

File: docs/project/2024/george.radu2504/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+
+# Smart lamp
+The project involves creating a smart lamp that responds to motion and ambient light levels. 
+
+:::info 
+
+**Author**: Radu George-Alexandru
```
Comment (alexandruradovici) [inline docs/project/2024/george.radu2504/index.md:7]:
```suggestion
**Author**: Radu George-Alexandru \
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/george.radu2504/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+
+# Smart lamp
+The project involves creating a smart lamp that responds to motion and ambient light levels. 
+
+:::info 
+
+**Author**: Radu George-Alexandru \
+
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-George2543
```
Comment (alexandruradovici) [inline docs/project/2024/george.radu2504/index.md:9]:
```suggestion
**Author**: Radu George-Alexandru \
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-George2543
```

File: docs/project/2024/george.radu2504/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+
+## Motivation
+
+I choose to persue this smart lamp project because I was thinking that one of the most important facilities of a smart house are the lights. This project tackles energy efficiency by innovating traditional lighting with a Raspberry Pi Pico W and motion/light sensors. It's a technical challenge that excites me, merging hardware and software seamlessly, so that I can craft a smart home solution that contributes to a mindful energy consumption.
+
+## Architecture 
+
+
+![Rasberry Pi Pico W (5)](https://github.com/George2543/upb-fils-ma.github.io/assets/154756379/599a908e-feed-4ca3-8aa4-25665cae1950)
```
Comment (alexandruradovici) [inline docs/project/2024/george.radu2504/index.md:25]: Please use the picture from this repository.

File: docs/project/2024/george.radu2504/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+| [embassy-time](https://github.com/embassy-rs/embassy/tree/main/embassy-time) | Time Library | Used for creating timing functions |
+| [embassy-rp](https://github.com/embassy-rs/embassy/tree/main/embassy-rp) | RP2040 pheripherals | Accesing the pheripherals of the microcontroller  |
+
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [YouTube](https://www.youtube.com/watch?v=l9MAZu7yvN4&ab_channel=Shilleh)
+2. [YouTube](https://www.youtube.com/watch?v=OapZNPaKrmA&ab_channel=ElectronicsUniversity)
```
Comment (alexandruradovici) [inline docs/project/2024/george.radu2504/index.md:104]: Write the video titles in the links title.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**